### ISSUE TITLE
Update README.md for hacking case

### ISSUE THREAD

**Issue by phracek** [fedora-infra/the-new-hotness#88]
I would suggest to update README.md file in case that fedmsg-replay does not work.

What to do with the-new-hotness in case that msg_id is send by fedmsg-dg-replay but fedmsg-hub does not receive it?

Like in my case:
E.g. https://apps.fedoraproject.org/datagrepper/id?id=2015-0a6ec8b3-eb5e-4921-b061-32cb0dcd8cdc&is_raw=true&size=extra-large
2015-0a6ec8b3-eb5e-4921-b061-32cb0dcd8cdc

~~~
(my-env)[root@rebase-helper-upstream-monitoring the-new-hotness]# fedmsg-dg-replay --msg-id 2015-0a6ec8b3-eb5e-4921-b061-32cb0dcd8cdc
Retrieving '2015-0a6ec8b3-eb5e-4921-b061-32cb0dcd8cdc'
Broadcasting '2015-0a6ec8b3-eb5e-4921-b061-32cb0dcd8cdc'
OK.
(my-env)[root@rebase-helper-upstream-monitoring the-new-hotness]# 

~~~
Also switch '/etc/fedmsg.d/base.py' from 'dev' to 'prod' doesn't help.

Any other idea?
I run the-new-hotness as root user.

**Comment by ralphbean** [comment]
Were you able to see the replayed message in another terminal running ``fedmsg-tail --really-pretty`` ?

**Comment by phracek** [comment]
Unfortunately not. fedmsg-tail --pretty doesn't show a message.

Meanwhile finally I am able to execute rebase-helper with upstream monitoring.
Scratch builds are going well. I will send a new PR for it.

**Comment by ralphbean** [comment]
Cool!

For replayed messages, it seems like either 1) fedmsg-relay must not be running or 2) the ports it wants are blocked by firewalld/iptables?

**Comment by phracek** [comment]
Well
ad2) 
~~~
[root@rebase-helper-upstream-monitoring ~]# systemctl list-units | grep firewall
[root@rebase-helper-upstream-monitoring ~]# systemctl list-units | grep iptables
[root@rebase-helper-upstream-monitoring ~]# 
~~~
ad1) fedmsg-relay was running. I turned off but grep over tail.txt (fedmsg-tail > tail.txt 2>&1)
doesn't show the message.

**Comment by phracek** [comment]
This is closed by commit  https://github.com/fedora-infra/the-new-hotness/commit/d3805d30007184eaec091c35d849d3967a568224
